db/migrate/20200303115024_optin_facts_and_reports.rb

```diff
@@ -0,0 +1,11 @@
+class OptinFactsAndReports < ActiveRecord::Migration[5.2]
+  def self.up
+    Setting.find_by(name: 'enable_facts_storage').update(value: FactValue.unscoped.any?)
```

Comment:
This causes tests to fail. When migrations are ran on new db, settings does not exist yet. So we run `nil.update`. We need to skip updating it if it does not exist.